app.js

```diff
@@ -2,44 +2,33 @@ angular.module('PathOfDamage', [])
 .controller('Damage', function ($scope, DataService) {
   $scope.sections = DataService.getSections();
 
-  $scope.hits = [{hit: 100}, {hit: 500}, {hit: 1000}, {hit: 2000}, {hit: 3000},
-    {hit: 4000}, {hit: 5000}, {hit: 7500}, {hit: 10000}];
+  $scope.hits = [{hit: 100}, {hit: 500}, {hit: 1000}, {hit: 2000}, {hit: 3000}, {hit: 5000}, {hit: 7500}, {hit: 10000}];
   $scope.resistance = 75;
-  $scope.healthPool = 7000;
+  $scope.healthPool = 5000;
```

Comment:
with inc nerfs to ES I expect a lot more life users, and average life pool for early endgame is fairly close to 5k ish range. I dunno, usually ES doesn't have to care that much about hit damage, just leeching sustain?